plugins/Carbonix/UI/LandingPlanForm.cs

```diff
@@ -589,6 +589,9 @@ private void but_accept_Click(object sender, EventArgs e)
             var sign = rad_loitcw.Checked ? 1 : -1;
             plugin.Host.AddWPtoList(MAVLink.MAV_CMD.DO_LAND_START, 0, 0, 0, 0, loiter_point.Lng, loiter_point.Lat, (double)num_transit_alt.Value);
 
+            //Turn off camera triggers
+            plugin.Host.AddWPtoList(MAVLink.MAV_CMD.DO_SET_CAM_TRIGG_DIST, 0, 0, 1, 0, 0, 0, 0);

```

Comment:
```suggestion
            plugin.Host.AddWPtoList(MAVLink.MAV_CMD.DO_SET_CAM_TRIGG_DIST, 0, 0, 0, 0, 0, 0, 0);

```

A `1` in `param3` makes it trigger once in addition to setting the distance.